### PR TITLE
Microsoft Ignite and Open Census

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
New blog post.

### CODE REVIEW

**Comment by songy23** [comment]
> The good news is that everyone that needs to sign a CLA (the pull request submitter and all commit authors) have done so. Everything is all good there.

Manually verified.

**Comment by songy23** [comment]
Looks like the image is not shown correctly on the web page:
ms-ignite

Could you move the images to directory static/images/?

File: content/articles/ms-ignite-2018.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+During Ignite's keynote, Microsoft announced support for Azure Kubernetes Service (AKS) Virtual Nodes. Virtual Nodes allow applications to scale more quickly, in a matter of mere minutes. The demonstration included a highlight of Azure Monitor [Live
+Metrics Stream](https://docs.microsoft.com/azure/application-insights/app-insights-live-stream).
+
+[![Microsoft Ignite 2018 keynote AKS Virtual Nodes with Application
+Insights and Open Census](ms-ignite-2018-images/ms-ignite-kyenote.gif)](https://mediastream.microsoft.com/events/2018/1809/Ignite/player/tracks/track2.html?start=17300)
+
+The keen eyed among you may have noticed that this demo is powered by [OpenCensus](https://github.com/Azure-Samples/virtual-node-autoscale)!
+
+![OpenCensus in Ignite Keynote](ms-ignite-2018-images/keynote-demo-app-uses-open-census.png)
```
Comment (songy23) [inline content/articles/ms-ignite-2018.md]: Please move this image to `static/images/`, then update the path to `/images/keynote-demo-app-uses-open-census.png`

File: content/articles/ms-ignite-2018.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+### Introduction
+Just three months ago [we announced that Microsoft joined the OpenCensus project](https://open.microsoft.com/2018/06/13/microsoft-joins-the-opencensus-project/), and last week at [Microsoft Ignite](https://www.microsoft.com/ignite) we were able to share a few more details about how OpenCensus will be leveraged and advanced.
+
+### OpenCensus spotlight
+
+During Ignite's keynote, Microsoft announced support for Azure Kubernetes Service (AKS) Virtual Nodes. Virtual Nodes allow applications to scale more quickly, in a matter of mere minutes. The demonstration included a highlight of Azure Monitor [Live
+Metrics Stream](https://docs.microsoft.com/azure/application-insights/app-insights-live-stream).
+
+[![Microsoft Ignite 2018 keynote AKS Virtual Nodes with Application
+Insights and Open Census](ms-ignite-2018-images/ms-ignite-kyenote.gif)](https://mediastream.microsoft.com/events/2018/1809/Ignite/player/tracks/track2.html?start=17300)
```
Comment (songy23) [inline content/articles/ms-ignite-2018.md]: Same for this image.

File: content/articles/ms-ignite-2018.md (hunk 1 above)
Comment (SergeyKanzhelev) [reply content/articles/ms-ignite-2018.md]: Thank you for detailed instructions. Fixed. Is there a way I can check it?

I know other systems (like octopress) has the same issue - markdown-valid path for images wouldn't work when rendered. [Docfx](https://dotnet.github.io/docfx/) doesn't have this issue so I get used to it =)

**Comment by songy23** [comment]
Thanks, looks good now:
ms-ignite png

File: content/articles/ms-ignite-2018.md (hunk 1 above)
Comment (songy23) [reply content/articles/ms-ignite-2018.md]: For a quick check, you can build the website locally using Hugo and navigate to the new page. The instructions are https://github.com/census-instrumentation/opencensus-website#contributing, then the updated opencensus.io will be served at localhost:1313.

**Pull request merged**